lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -45,6 +45,17 @@ describe('ShareDialogComponent', () => {
 
     const getShareToggleLinkedClasses = (): DOMTokenList => fixture.nativeElement.querySelector(shareToggleId).classList;
 
+    const fillInDatepickerInput = (value: string) => {
+        const input = fixture.debugElement.query(By.css('.adf-share-link__input')).nativeElement;
+        input.value = value;
+        input.dispatchEvent(new Event('input'));
+        input.dispatchEvent(new Event('blur'));
+        fixture.detectChanges();
+    };
+
+    const clickExpireToggleButton = () => fixture.nativeElement.querySelector('mat-slide-toggle[data-automation-id="adf-expire-toggle"] label')
```

Comment:
Could you reuse that in 333rd line as there is the same logic?

Reply:
adressed, thanks